app/Http/Controllers/BillingController.php

```diff
@@ -41,17 +43,20 @@ class BillingController extends Controller
     private AccountBillingController $accountBillingController;
     private SystemController $systemController;
     private AccountController $accountController;
+    protected $contactService;
 
-    public function __construct()
+    public function __construct(ContactService $contactService)
```

Comment:
This looks like it needs to either be defaulted or created inside the constructor without an argument like the other fields. There are a few references flagged at the bottom like `Class App\Http\Controllers\BillingController constructor invoked with 0 parameters, 1 required.` where the argument isn't being passed. There are also a number of navigation calls like `->action([BillingController::class, 'index'])` that are probably impacted by this.